lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -107,6 +107,8 @@ class AnnotationReader implements Reader
         // PHP CodeSniffer
         'codingStandardsIgnoreStart' => true,
         'codingStandardsIgnoreEnd' => true,
+        // PHPStan
+        'template' => true, 'implements' => true, 'extends' => true,
```

Comment:
These annotations are too widespread to be ignored, specifically talking about existing applications that rely on things such as `@template("mystuff.twig")`, for example.

Usage of `@phpstan-template` (already ignored by the parser) is to be endorsed instead